src/System.Management.Automation/engine/NullString.cs

```diff
@@ -6,7 +6,7 @@ namespace System.Management.Automation.Language
     /// <summary>
     /// This type is introduced to provide a way to pass null into a .NET method that has a string parameter.
     /// </summary>
-    public class NullString
+    public sealed class NullString
```

Comment:
Causes test failure due to public API change:

https://dev.azure.com/powershell/PowerShell/_build/results?buildId=45285&view=ms.vss-test-web.build-test-results-tab&runId=1310450&resultId=100810&paneView=debug

Reply:
It is in SDK. It can be a huge breaking change. Why do we really need the PS's changes?

Reply:
reverted

Reply:
@iSazonov Maybe a change to the API would not be so bad. In fact subclassing `NullString` would be a bad idea so maybe we should disallow this?

Reply:
Maybe. It is better to open new issue and discuss there (and get PowerShell Committee conclusion)